core/arch/arm/plat-ti/platform_config.h

```diff
@@ -28,87 +28,110 @@
 #ifndef PLATFORM_CONFIG_H
 #define PLATFORM_CONFIG_H
 
-#if defined(PLATFORM_FLAVOR_dra7xx) || defined(PLATFORM_FLAVOR_am57xx)
-
-#define DRAM0_BASE		0xbe000000
-#define DRAM0_SIZE		0x02000000
+#define DRAM0_BASE      0x80000000
+#define DRAM0_SIZE      0x80000000
 
-#ifdef CFG_WITH_PAGER
-#error Pager not supported on this platform
-#endif /*CFG_WITH_PAGER*/
+#if defined(PLATFORM_FLAVOR_dra7xx) || defined(PLATFORM_FLAVOR_am57xx)
 
 /* Location of protected DDR on the DRA7xx platform */
-#define TZDRAM_BASE		0xbe000000
-#define TZDRAM_SIZE		0x01c00000
+#define TZDRAM_BASE     0xbdb00000
+#define TZDRAM_SIZE     0x01c00000
```

Comment:
It should probably not be done in this patch, but I guess sooner or later you will probably want to carveout memory for SDP also.